index.html

```diff
@@ -4,314 +4,348 @@
+            </ul>
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+
+        <div class="card2">
+          <div class="card2001">
+            <img src="./image/card2.png" alt="some" />
+          </div>
+          <div class="c002">
+            <h3>Multi-Post Stories</h3>
+            <ul>
+              <li>CANOPY</li>
+              <li><img src="./image/image3.png" alt="count1" /></li>
               <li>Back End Dev</li>
-              <li><div class="dot"></div></li>
+              <li><img src="./image/image4.png" alt="count2" /></li>
               <li>2015</li>
             </ul>
-            <p class="pro">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
+            <p>
+              A daily selecion of privately personalized reads; no accounts or
+              sign up required.
             </p>
-            <ul class="language">
-              <li>HTML</li>
-              <li>CSS</li>
-              <li>JavaScript</li>
+            <ul class="lang">
+              <li><h5>html</h5></li>
+              <li><h5>css</h5></li>
+              <li><h5>javascript</h5></li>
             </ul>
-            <a href="#" class="button">See Project</a>
+            <button class="button" type="button">See Project</button>
           </div>
         </div>
-        <div class="works">
-          <img src="./images/Snapshoot Portfolio.png" alt="" />
-          <div class="card-work work">
-            <h3>
-              Multi-Post <br />
-              Stories
-            </h3>
-            <ul class="proj">
-              <li><a href="#">CANOPY</a></li>
-              <li><div class="dot"></div></li>
+
+        <div class="swapcard3">
+          <div class="card3001">
+            <img src="./image/image2.png" alt="" />
+          </div>
+          <div class="c002">
+            <h3>TONIC</h3>
+            <ul>
+              <li>CANOPY</li>
+              <li><img src="./image/image3.png" alt="count1" /></li>
               <li>Back End Dev</li>
-              <li><div class="dot"></div></li>
+              <li><img src="./image/image4.png" alt="count2" /></li>
               <li>2015</li>
             </ul>
-            <p class="pro">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
+            <p>
+              A daily selecion of privately personalized reads; no accounts or
+              sign up required.
             </p>
-            <ul class="language">
-              <li>HTML</li>
-              <li>CSS</li>
-              <li>JavaScript</li>
+            <ul class="lang">
+              <li><h5>html</h5></li>
+              <li><h5>css</h5></li>
+              <li><h5>javascript</h5></li>
             </ul>
-            <a href="#" class="button">See Project</a>
+            <button class="button" type="button">See Project</button>
```

Comment:
[Optional]

- [ ] This is marked as optional because the previous reviewer missed it but it's part of the requirements 👍 Kindly add button interaction styles to all your buttons as shown in Figma 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/208114438-1c0c3e54-c3f2-4c7d-8020-4fafbdf050d3.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/208114500-4a32246b-2ed8-48c0-a998-97bbd9d8e31f.png)